source/openapi3/split.yaml

```diff
@@ -4631,7 +4061,7 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Payout'
-          description: One or many Payouts
+          description: One Payout object only
```

Comment:
Are we moving away from the term _payout_?

Reply:
Yeh I'm trying to clean up Payout references as its confusing when we talk about Payments. But Payout should mean the same thing as a Payment, but sometimes Payouts have been used in the context of Payment Request which is madness

Reply:
> sometimes

You should have a look in the codebase 😆